webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -445,6 +446,7 @@ var CURRENT_GROUP_DATA={};
         var USE_BRANDING = typeof appOptions.useBranding  == 'boolean'? appOptions.useBranding : true;
         var POPUP_WIDGET = appOptions.popupWidget;
         w.MCK_OL_MAP = new Array();
+        var VOICE_INPUT_ENABLED = appOptions.enableVoiceInput;
```

Comment:
@Suraj-Kumar in options name, we can keep it simple as voiceInput. enable or disable is what its value will speak.

Reply:
wanted to keep it in sync with other parameters.  other parameters use property true/false. shall  I change it?

Reply:
@Suraj-Kumar no, i mean keep the property same as voiceInput, same as how other properties are locShare, popUpWidget, attachment etc.
voiceInput: true/false

Reply:
ok

Reply:
done